packs/equipment-effects/effect-skinstitch-salve.json

```
@@ -4,13 +4,13 @@
     "name": "Effect: Skinstitch Salve",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Skinstitch Salve]</p>\n<hr />\n<p>Auomatically applied effects:</p>\n<ul>\n<li>gain a +2 item bonus to a Medicine check</li>\n</ul>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Skinstitch Salve]</p>\n<hr />\n<p>Auomatically applied effects:</p>\n<ul>\n<li>gain a +2 item bonus to a Medicine check</li>\n\n<li>improves a success on Treat Wounds or Battle Medicine to a critical success instead</li>\n</ul>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Skinstitch Salve]</p>\n<hr />\n<p>You gain a +2 item bonus to medicine checks and if you roll a success on Treat Wounds or Battle Medicine you get a critical success instead</p>"
```
That automatically applied effects text always irked me, really old format for us.